me.drewol.Unnamed-SDVX-Clone.metainfo.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>me.drewol.Unnamed-SDVX-Clone</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MIT</project_license>
+  <name>Unnamed SDVX Clone</name>
+  <summary>A game based on KShootMANIA &amp; SDVX</summary>
+  <description>
+    <p>
+        A game based on K-Shoot MANIA and Sound Voltex
+    </p>
+    <ul>
+      <li>Completely skinnable GUI</li>
+      <li>OGG/MP3 Audio streaming (with preloading for gameplay performance)</li>
+      <li>Uses KShoot charts (<code>*.ksh</code>) (1.6 supported)</li>
+      <li>Functional gameplay and scoring</li>
+      <li>Saving of local scores</li>
+      <li>Autoplay</li>
+      <li>Basic controller support</li>
+      <li>Changeable settings and key mapping</li>
+      <li>Supports new sound FX method (real-time sound FX) and old sound FX method (separate NOFX &amp; sound effected music files)</li>
+      <li>Song database cache for near-instant game startup (sqlite3)</li>
+      <li>Song database searching</li>
+      <li>Linux/Windows/macOS support</li>
+      <li>Song select UI/Controls to change HiSpeed and other game settings</li>
+    </ul>
+  </description>
+  <launchable type="desktop-id">me.drewol.Unnamed-SDVX-Clone.desktop</launchable>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://i3.ytimg.com/vi/1GCraT0ktrc/hqdefault.jpg</image>
+      <caption>Gameplay screenshot showcasing default colors</caption>
+    </screenshot>
+    <screenshot>
+      <image>https://i3.ytimg.com/vi/kP1tD6bkPa4/hqdefault.jpg</image>
+      <caption>Gameplay screenshot showcasing a vertical resolution</caption>
+    </screenshot>
+    <screenshot>
+      <image>https://i3.ytimg.com/vi/_g9Xv5RDwa0/hqdefault.jpg</image>
```

Comment:
Can you put some actual screenshots in the upstream repo and link them here? The youtube screenshots are pretty bad in terms of quality.

Reply:
I'm not the upstream maintainer, so I'm trying to avoid pushing pull requests on them wherever I can. Would adding them to this repo be okay?

Reply:
Upstream maintainer here. You should be able to change `hqdefault` to `maxresdefault` for better quality. The vertical resolution screenshot doesn't quite reflect how the vertical gameplay currently looks as those are quite old screenshots so I'd recommend a new screenshot or just skipping it.